public/index.php

```diff
@@ -6,16 +6,31 @@
 header("Content-Type: application/json");
 
 use Gacela\Framework\Gacela;
+use PhpLightning\Config\Backend\LnBitsBackendConfig;
+use PhpLightning\Config\LightningConfig;
 use PhpLightning\Lightning;
 
 $cwd = (string)getcwd();
+// $cwd = "/opt/lnaddress/demo/demo-lnaddress/";            // path to composer working directory (to be reviewed)
```

Comment:
@Chemaclass Please check if I'm using the right vocabulary here regarding "composer working directory"

Reply:
Thanks for the hint! However, I found a better and simpler way (developer experience, at least) to work on this. I will close this PR in favor of another one.